answers/javaScript.md

```diff
@@ -1,3 +1,5 @@
 ## JavaScript
 * `null` vs `undefined` - Null means there is 'no value.' Undefined represents a variable that has not been declared or not given a value.
 	* What does `type of null` output? It returns 'object.'
+* `==` vs `===` - == compares values but not data types. eg. 3 == "3" would be true. ie... it converts the values.
```

Comment:
This feels like it needs some line breaks and more styling to make it more readable.